tests/routes/test_container_service.py

```diff
@@ -0,0 +1,13 @@
+
+class TestContainerService:
+    def test_create_environment(self):
+        assert True
```

Comment:
Use the [flask test_client](https://flask.palletsprojects.com/en/1.1.x/testing/#the-testing-skeleton) to invoke the REST endpoint.

Reply:
done